airbyte-integrations/connectors/source-tiktok-marketing/integration_tests/streams.json

```diff
@@ -108,13 +108,6 @@
         "json_schema": {},
         "supported_sync_modes": ["full_refresh"]
       },
-      {
```

Comment:
This streams.json I don't think is actually used in our acceptance test YAML but just keeping this up to date regardless